README.md

```diff
@@ -5,13 +5,12 @@
 [![npm download](https://img.shields.io/npm/dm/authmosphere.svg?style=flat-square)](https://www.npmjs.com/package/authmosphere)
 [![npm version](https://img.shields.io/npm/v/authmosphere.svg?style=flat)](https://www.npmjs.com/package/authmosphere)
 
-## Project renaming
+## Introduction
 
-The project was renamed from `lib-oauth-tooling` to `authmosphere`. In the course of this renaming versioning was restarted at `0.1.0`. Currently the change is non-breaking from a code perspective. Version `1.0.0` is soon to be released, keep track of the progess in https://github.com/zalando-incubator/lib-oauth-tooling/issues/92, this release will most likely contain breaking changes.
+Authmosphere is a JavaScript library to support some common OAuth2 use cases.
```

Comment:
Maybe be more concrete than "some common"? Like:
__authmosphere__ (I would suggest to always write in lower case?) is a library to support OAuth2 workflows in JavaScript projects.